Jenkinsfile

```diff
@@ -0,0 +1,20 @@
+pipeline {
+    agent {
+        docker {
+            image 'magnonellie/amethyst-dependencies:stable'
+            label 'docker'
+        } 
+    }
+    stages {
+        stage('fmt') {
+            steps {
+                sh 'cargo fmt --all -- --check'
+            }
+        }
+        stage('test') {
+            steps {
+                sh 'cargo test --all'
```

Comment:
Can we add `--all-features`?

Reply:
Ah yes, that will happen this isnt ready yet :) opened this for testing purposes.